packaging/nnstreamer.spec

```diff
@@ -253,6 +253,10 @@ BuildConflicts: libarmcl-release
 BuildRequires:  json-glib-devel
 %endif
 
+%if 0%{?pytorch_support}
```

Comment:
can you also add the same for caffe2 support? for some reason, that is also missing.

Reply:
I will.